### PR TITLE
Fuzz tests

### PR DESCRIPTION
**Pull request by stoeffel**
### WIP :rotating_light:

this is still WIP. I just want to give you an early heads up. Would be cool to get your feedback, so I don't move in a direction you don't like.
## What Changed ✏️
- updates to latest elm-test
- adds fuzz tests

### CODE REVIEW

File: tests/Tests.elm [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                \z b f a ->
+                    Expect.equal ((List.reverse b) ++ [ f ] ++ a) <| toList z
+            ]
+        ]
+
+
+updating : Test
+updating =
+    let
+        negtive =
```
Comment (stoeffel) [inline tests/Tests.elm:76]: should be `modify`. This happens when you watch netflix at the same time :laughing:

**Comment by stoeffel** [comment]
I will propaply do "moving around" not as fuzz-tests.

**Comment by wernerdegroot** [comment]
Nice! 

Should we put the old "moving around"-tests back? It appears we've lost some coverage now.

**Comment by stoeffel** [comment]
> Should we put the old "moving around"-tests back? It appears we've lost some coverage now.

Yes, was going to add that. This was still WIP 😄

**Pull request merged**